### ISSUE TITLE
Cannot type the branch name in Checkout branch

### ISSUE THREAD

**Issue by sharwell** [gitextensions/gitextensions#5415]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

The **Select branch** controls is a read-only drop down. Typing only works if you type the branch name without mistakes and the keystroke latency never exceeds the reset timeout.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**

The **Select branch** should not be read-only. Users should be able to *type* the branch name.

**Environment you encounter the issue:**

- GitExtensions version: 3.00.00.3336Dev (c6155)
- GIT version: 
- OS version: 
- .NET version: 

**Did this work in previous version of GitExtensions (which)?**

Yes, this regression was introduced by #5184.

**Comment by vbjay** [comment]
I concur.  I tried it.  Typing "fea" selects the first feature/... branch in the combobox.  it should be a filter.  I type fea and branches that contain or start with fea show up.  It can act like the combobox in windows explorer's path.

It should be an autocomplete dropdown.  I typed b after the \ in the current path and then hit down arrow.

![image](https://user-images.githubusercontent.com/3629489/45264726-17b44780-b40f-11e8-8783-f87bc30916d9.png)
